### ISSUE TITLE
Missing capsule name in SQL code generation

### ISSUE THREAD

**Issue by rfrasier-mlp** [Roenbaeck/anchor#42]
This is for the production version (0.97.6) of the online modeler.  For some reason, the capsule name is no longer being correctly inserted.  This is from a new model with two attributes and no changes from default settings.

Please see below picture:

![issue with sql code generation](https://cloud.githubusercontent.com/assets/14884371/10355886/6bf7f504-6da3-11e5-8c74-9ade3cf2951e.png)

**Comment by Roenbaeck** [comment]
I cannot reproduce this, I tried 0.97, 0.98, and 0.99. That particular value is picked up from whatever is under the "Defaults" menu and "Capsule" setting. Can you first make sure that the field is not blank, and if not provide an XML file of the model?

**Comment by rfrasier-mlp** [comment]
The test version 0.99 works fine for me.  It's only the production version 0.98 that has this issue.  I'm using Firefox version 32.0.3 if that helps.  Are there any other environment details that might assist?

I ensured that the capsule was set to `dbo` and have the same issue when tested with several other models that I have made in the past.  It seems this has only cropped up in the last week, perhaps only since the latest update?

The XML looks fine to me but here it is for my simple model:

```XML
<schema format="0.97" date="2015-10-09" time="07:56:54">
  <metadata changingRange="datetime" encapsulation="dbo" identity="int" metadataPrefix="Metadata" metadataType="int" metadataUsage="false" changingSuffix="ChangedAt" identitySuffix="ID" positIdentity="int" positGenerator="true" positingRange="datetime" positingSuffix="PositedAt" positorRange="tinyint" positorSuffix="Positor" reliabilityRange="tinyint" reliabilitySuffix="Reliability" reliableCutoff="1" deleteReliability="0" reliableSuffix="Reliable" partitioning="false" entityIntegrity="true" restatability="false" idempotency="true" naming="improved" positSuffix="Posit" annexSuffix="Annex" chronon="datetime2(7)" now="sysdatetime()" dummySuffix="Dummy" versionSuffix="Version" statementTypeSuffix="StatementType" checksumSuffix="Checksum" businessViews="false" databaseTarget="SQLServer" temporalization="crt"/>
  <anchor mnemonic="CO" descriptor="Company" identity="int">
    <metadata capsule="dbo" generator="true"/>
    <attribute mnemonic="NAM" descriptor="Name" identity="int" timeRange="datetime" dataRange="varchar(42)">
      <metadata capsule="dbo" generator="true" restatable="false" idempotent="true"/>
      <layout x="653.48" y="660.55" fixed="false"/>
    </attribute>
    <attribute mnemonic="PRI" descriptor="Price" identity="bigint" timeRange="datetime2(7)" dataRange="decimal(17,7)">
      <metadata capsule="dbo" generator="true" restatable="true" idempotent="false"/>
      <layout x="718.56" y="712.72" fixed="false"/>
    </attribute>
    <layout x="715.65" y="675.69" fixed="false"/>
  </anchor>
</schema>
```

Many thanks for your help!

**Comment by Roenbaeck** [comment]
I could only get my hands on Firefox 41.0.1, but for both that and Chrome version 45.0.2454.101 m (64-bit) the model you posted does have the `dbo` in place in v0.98 of the modeler. Could it be some kind of browser cache issue?

**Comment by rfrasier-mlp** [comment]
You're right, it looks like it was a browser cache issue with Firefox.  I cleared the cache and I have tested loading old models and creating new ones using CRT and UNI and there are now no missing capsule names.  This issue can be closed now I believe, but I will certainly monitor my cache to see if something triggers this behavior in the future and perhaps play with cache management settings.  Many thanks!
